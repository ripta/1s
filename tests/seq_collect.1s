"seq" {LOAD}

"collect-into"
3 [ 20 * ] [ 0 0 0 0 0 ] collect-into i

"collect-into*"
[ 0 0 0 0 0 ] 3 [ 20 * ] collect-into* i

"collect"
0 5 [ 6 + ] [ 0 0 0 0 0 ] collect

"0,collect"
5 [ 6 + ] [ 0 0 0 0 0 ] 0,collect
