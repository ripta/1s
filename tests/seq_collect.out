Result:
  "collect-into"
  [ 0 0 0 60 0 ]
  "collect-into*"
  [ 0 0 0 60 0 ]
  "collect"
  [ 6 7 8 9 10 ]
  "0,collect"
  [ 6 7 8 9 10 ]
